geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2018 Esri, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/**
+ * Utilities For Reading Various Flat Files
+ */
+
+package com.esri.geoportal.commons.doc;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Properties;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.tika.exception.TikaException;
+import org.apache.tika.metadata.TikaCoreProperties;
+import org.apache.tika.metadata.Metadata;
+import org.apache.tika.parser.AutoDetectParser;
+import org.apache.tika.parser.ParseContext;
+import org.apache.tika.parser.Parser;
+import org.apache.tika.sax.BodyContentHandler;
+import org.w3c.dom.Document;
+
+import com.esri.geoportal.commons.meta.AttributeUtils;
+import com.esri.geoportal.commons.meta.MapAttribute;
+import com.esri.geoportal.commons.meta.util.WKAConstants;
+import com.esri.geoportal.commons.meta.xml.SimpleDcMetaBuilder;
+import com.esri.geoportal.commons.utils.XmlUtils;
+
+public class DocUtils {
+	
+	// Local Files For Testing Porpoises 
+	static String[] file_set = {
```

Comment:
Are these hard coded paths necessary here?

Reply:
Removed in f63aa55